laravel/app/Console/Commands/AutoDeliveryCommand.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Models\OrderItem;
+use App\Models\Inventory;
+use App\Models\GoogleSheet;
+use Carbon\Carbon;
+
+class AutoDeliveryCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'command:auto_delivery';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Command description';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $sheets = GoogleSheet::OrderItem();
+
+        //$service = new \Google_Service_Sheets($client);
+
+
+        $sheet_id = '1DRe3JKouPvmXoosZXlhXcNOGnALHO61J39QTItwAMHc';
+        $valueInputOption = "USER_ENTERED";
+        $ship_date = '2020-09-19';
+        $range = 'A2';
+        
+        $order_indexes = OrderItem::AutoDeliveryIndex($ship_date)->get();
+
+        foreach ($order_indexes as $order) {
+
+            Inventory::ShipmentList($order)->post();
```

Comment:
scopeの内容は別のところでコメントした内容に書き換えて、ここはfirstで1件だけ取得するようにしましょう
```suggestion
            $inventory = Inventory::ShipmentList($order)->first();
```

Reply:
$inventoryが取得出来るようになったら、下記のような更新方法もあるので
まずはこれで更新が出来るか試してみましょう
```
            $ship_arranged = \Config::get('const.Temporaries.ship_arranged');
            $inventory->order_item_id = $order->id;
            $inventory->ship_date = $order->ship_date;
            $inventory->status = $ship_arranged;
            $inventory->save();
```